web-components/src/components/form/BridgeForm.tsx

```diff
@@ -66,7 +70,7 @@ const BridgeForm = ({
       {({ values, submitForm, isSubmitting, isValid, submitCount, status }) => (
         <Form>
           <Field
-            name="chain"
+            name="target"
             label="Chain"
             component={SelectTextField}
             options={[{ label: 'Polygon', value: 'polygon' }]}
```

Comment:
@flagrede I'm extracting a constant with the allowed chains for bridge. Currently only one item for Polygon, and the initial values of the form read from this first option